tests/phpunit/integration/tests/User/Preferences.php

```diff
@@ -53,15 +61,21 @@ public static function wpSetUpBeforeClass( $factory ) {
 	}
 
 	public function set_up() {
-		$up = new \Google\Web_Stories\User\Preferences();
-		$up->register();
 		parent::set_up();
+
+		$this->instance = new \Google\Web_Stories\User\Preferences();
 	}
 
 	public function tear_down() {
+		delete_user_meta( self::$admin_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
+		delete_user_meta( self::$admin_id, \Google\Web_Stories\User\Preferences::OPTIN_META_KEY );
+		delete_user_meta( self::$author_id, \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
+		delete_user_meta( self::$author_id, \Google\Web_Stories\User\Preferences::OPTIN_META_KEY );
+
 		unregister_meta_key( 'user', \Google\Web_Stories\User\Preferences::OPTIN_META_KEY );
 		unregister_meta_key( 'user', \Google\Web_Stories\User\Preferences::ONBOARDING_META_KEY );
 		unregister_meta_key( 'user', \Google\Web_Stories\User\Preferences::MEDIA_OPTIMIZATION_META_KEY );
```

Comment:
```suggestion
		delete_user_meta( self::$admin_id, $this->instance::MEDIA_OPTIMIZATION_META_KEY );
		delete_user_meta( self::$admin_id, $this->instance::OPTIN_META_KEY );
		delete_user_meta( self::$author_id, $this->instance::MEDIA_OPTIMIZATION_META_KEY );
		delete_user_meta( self::$author_id, $this->instance::OPTIN_META_KEY );

		unregister_meta_key( 'user',$this->instance::OPTIN_META_KEY );
		unregister_meta_key( 'user',$this->instance::ONBOARDING_META_KEY );
		unregister_meta_key( 'user', $this->instance::MEDIA_OPTIMIZATION_META_KEY );
```

Does this make sense?

Reply:
I don't think we really need to use the instance here for the test cleanup, but I'm indifferent.